src/lfortran/semantics/comptime_eval.h

```
@@ -147,9 +147,7 @@ struct IntrinsicProcedures {
                 return cb(al, loc, expr_args, compiler_options);
             }
         } else {
-            throw SemanticError("Intrinsic function '" + name
-                + "' compile time evaluation is not implemented yet",
-                loc);
+            return nullptr;
```

What is the implication of this change --- does it silence an error? We might not want to do that, but I don't understand the context yet.

I think this is old `comptime_eval` that we are slowly getting rid of, correct?

---

I think it shall never be an error just nullptr when compile time value is not present. Also this is old `comptime_eval` and I do not think we use it that much nowadays.

---

I think this is fine, this is the old implementation.